stdlib/os/__init__.pyi

```diff
@@ -575,16 +575,18 @@ if sys.platform != "win32":
     # Unix only
     def fchmod(fd: int, mode: int) -> None: ...
     def fchown(fd: int, uid: int, gid: int) -> None: ...
-    if sys.platform != "darwin":
-        def fdatasync(fd: FileDescriptorLike) -> None: ...  # Unix only, not Mac
     def fpathconf(__fd: int, __name: str | int) -> int: ...
     def fstatvfs(__fd: int) -> statvfs_result: ...
     def get_blocking(__fd: int) -> bool: ...
     def set_blocking(__fd: int, __blocking: bool) -> None: ...
     def lockf(__fd: int, __command: int, __length: int) -> None: ...
     def openpty() -> tuple[int, int]: ...  # some flavors of Unix
     if sys.platform != "darwin":
-        def pipe2(flags: int) -> tuple[int, int]: ...  # some flavors of Unix
+        def fdatasync(fd: FileDescriptorLike) -> None: ...
+        if sys.version_info >= (3, 10):
+            def pipe2(__flags: int) -> tuple[int, int]: ...
+        else:
+            def pipe2(flags: int) -> tuple[int, int]: ...
```

Comment:
At least for Python 3.9, this is also positional only.

Reply:
I forgot that stubtest checks positional-only-correcntess only on 3.10.